server/src/main/java/org/opensearch/indices/IndicesService.java

```diff
@@ -1746,14 +1746,21 @@ private BytesReference cacheShardLevelResult(
      *
      * @opensearch.internal
      */
-    static final class IndexShardCacheEntity extends AbstractIndexShardCacheEntity {
-        private static final long BASE_RAM_BYTES_USED = RamUsageEstimator.shallowSizeOfInstance(IndexShardCacheEntity.class);
+    public final class IndexShardCacheEntity extends AbstractIndexShardCacheEntity {
+        private final long BASE_RAM_BYTES_USED = RamUsageEstimator.shallowSizeOfInstance(IndexShardCacheEntity.class);
```

Comment:
Why are we removing static keyword for this inner class? Is it for access `indices` member within IndicesService?

Reply:
Yes it is to access indices member.